api/src/main/java/org/openmrs/module/appointments/model/AppointmentPriority.java

```diff
@@ -0,0 +1,13 @@
+package org.openmrs.module.appointments.model;
+
+public enum AppointmentPriority {
+    AsNeeded("AsNeeded"), Routine("Routine"), Emergency("Emergency"), Invalid("Invalid");
```

Comment:
1. Apologies, Invalid priority is not in use and we removed it later. Updated the PR
2. For changing the priority codes as of FHIR model now would require some more time since it might need changes from the frontend as well. So we created a card in the backlog to track it [BAH-3637](https://bahmni.atlassian.net/browse/BAH-3637) and will be taken up later.